applications/StructuralMechanicsApplication/custom_elements/spring_damper_element.hpp

```diff
@@ -36,16 +36,16 @@ namespace Kratos
 ///@{
 
 /// Concentrated nodal for 3D and 2D points
-
-class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) SpringDamperElement3D2N 
+template<std::size_t TDim, std::size_t TNumNodes>
```

Comment:
why the template on the number of nodes?

I am quite sure this is always 2 no?

Reply:
yes you are right